treewide: remove rob

AndersonTorres opened the pull request:
Since theey is inactive.

## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

AndersonTorres commented on pkgs/development/libraries/opentracing-cpp/default.nix:
```diff
@@ -15,7 +21,6 @@ stdenv.mkDerivation rec {
     homepage = "https://opentracing.io";
     license = lib.licenses.asl20;
-    maintainers = with lib.maintainers; [ rob ];
+    maintainers = [ ];
```
This package should  be deleted.

SuperSandro2000 replied:
It is a dependency of a nginx module so we cannot just delete it.

SuperSandro2000 reviewed (commented):
I am not a fan that the same commit also did a bunch of reformatting. Also why weren't they also remove from the maintainers list?

AndersonTorres committed:
python3Packages.uptime: remove rob from maintainers

AndersonTorres committed:
cowsay: remove rob from maintainers

AndersonTorres committed:
cowsay: remove nested with from meta.maintainers

- Also, use a static changelog link.
- Further, inherit platforms from perl

AndersonTorres committed:
opentracing-cpp: remove rob from maintainers

AndersonTorres committed:
opentracing-cpp: nixfmt-rfc-style

AndersonTorres committed:
opentracing-cpp: migrate to by-name

AndersonTorres commented:
> I am not a fan that the same commit also did a bunch of reformatting.

The formatting is required. But since there are too few packages here, I will split  the things.

> Also why weren't they also remove from the maintainers list?

I feel this is not needed. Also it is a bit problematic for packages maintained by multiple silently-retired maintainers. It is better to do this in another round.

AndersonTorres committed:
htop-vim: migrate to by-name

AndersonTorres committed:
htop: migrate to by-name

AndersonTorres committed:
htop-nvim: rewrite

- nixfmt-rfc-style
- remove rec
- fix version
- remove references to by-name
- remove nested with

AndersonTorres committed:
htop: remove rob from maintainers

AndersonTorres committed:
htop: rewrite

- nixfmt-rfc-style
- remove references to pname
- remove nested with
- use static changelog link

AndersonTorres committed:
htop: finalAttrs design pattern

bendlas commented:
Closing these removal PRs, since they are not grounded within community consensus. See https://github.com/NixOS/nixpkgs/pull/337478#issuecomment-2310779183

Don't know the GH username of rob, but be sure that we value your involvement and any time you may have to spare, helping with the packages you're maintaining and others as well.

SuperSandro2000 commented:
it is @rbvermaa and that account is super inactive, so we are just pretending that the packages are maintained tbh
